src/search/PostList.tsx

```diff
@@ -0,0 +1,26 @@
+import { FunctionComponent } from 'react';
+
+interface PostTitle {
+  title: string;
+}
+interface PostListProps {
+  postTitles: PostTitle[];
+  selectedSearchValue: string;
+  inputSearchValue: string;
+}
+
+const PostList: FunctionComponent<PostListProps> = ({
+  postTitles,
+  selectedSearchValue,
+  inputSearchValue,
+}) => {
+  console.log(postTitles);
+  return (
+    <ul>
+      {postTitles.map((postTitle) => (
+        <li>{postTitle.title}</li>
```

Comment:
li 항목에 key값 꼭 추가해주세용~ㅎㅎ
그리고 아래처럼 변경할 수도 있을 거 같아요!


```suggestion
      {postTitles.map(({title}) => (
        <li>{title}</li>
```